docs/amber_script.md

```diff
@@ -555,6 +555,11 @@ contain image attachment content, depth/stencil content, uniform buffers, etc.
   # pipelines.
   BIND BUFFER {buffer_name} AS depth_stencil
 
+  # Attach |buffer_name| as a multisample resolve target. Each color attachment
+  # with more than one sample must have a corresponding single sample resolve
```

Comment:
When you say "must have", does this mean your change might break existing tests, including those in Vulkan CTS?

Reply:
I should probably reword this. I wasn't seeing a validation error without these so it should be still valid if I don't add the resolve images. Not that there's any point using multisample images without resolve as the results cannot be copied out of GPU memory with a transfer queue. Previously the multisample Amber usage in CTS has been compute shader tests where the results are checked inside the same shader where the samples get written. So nothings is getting broken.

I will change the text to something like "The order of resolve target images match with the order of color buffers that have more than one sample."